src/aiida_quantumespresso/workflows/pw/base.py

```diff
@@ -81,10 +81,10 @@ def define(cls, spec):
         spec.exit_code(211, 'ERROR_INVALID_INPUT_AUTOMATIC_PARALLELIZATION_UNRECOGNIZED_KEY',
             message='Unrecognized keys were specified for `automatic_parallelization`.'
                     'This exit status has been deprecated as the automatic parallellization feature was removed.')
-        spec.exit_code(300, 'ERROR_UNRECOVERABLE_FAILURE',
+        spec.exit_code(300, 'ERROR_UNRECOVERABLE_FAILURE [deprecated]',
             message='The calculation failed with an unidentified unrecoverable error.')
         spec.exit_code(310, 'ERROR_KNOWN_UNRECOVERABLE_FAILURE',
-            message='The calculation failed with a known unrecoverable error.')
+            message='The calculation failed with a known unrecoverable error. [deprecated]')
```

Comment:
This exit code is not deprecated, is it?

Reply:
That should be fine , known unrecoverable , means (at least to me) , there is a change or modification applied to avoid the error and those will still be active no ?

Reply:
Yeah, you are right (There were too many of them :D )